src/components/Form/Form.tsx

```diff
@@ -66,22 +157,349 @@ export default function Form({ setFormData }: IFormProps) {
+                  "&.Mui-selected": {
+                    backgroundColor: "#747bff",
+                    color: "white",
+                  },
+                  "& .MuiFormControl-root": {
+                    lineHeight: "1em",
+                  }, //!
+                },
+              }}
+            >
+              <InputLabel id="TypeOfAI-select-standard-label">
+                Type of AI
+              </InputLabel>
+              <Select
+                labelId="TypeOfAI-select-standard-label"
+                id="TypeOfAI-select-standard"
+                value={TypeOfAI}
+                onChange={onTypeChange}
+                label="Type of AI"
+                sx={{
+                  "& .MuiSelect-select": {
+                    fontSize: "1.2em",
+                  },
+                  "& .MuiFormControl-root": {
+                    lineHeight: "1em",
+                  }, //!
+                }}
+              >
+                <MenuItem value="">
+                  <em>None</em>
+                </MenuItem>
+                <MenuItem value={"NPC"}>NPC</MenuItem>
+                <MenuItem value={"Neural Network"}>Neural Network</MenuItem>
+                <MenuItem value={"Function"}>Function</MenuItem>
+              </Select>
+            </FormControl>
+          </div>
+          <div className="form-group">
+            <label>Rate AI intelligence</label>
+
+            <Box sx={{ width: "100%" }}>
+              <Grid container spacing={1} alignItems="center">
+                <Grid item xs>
+                  <Slider
+                    aria-label="Volume"
+                    value={typeof rangeValue === "number" ? rangeValue : 0}
+                    onChange={onRateSliderChange}
+                    aria-labelledby="input-slider"
+                    sx={{
+                      width: "95%",
+                      margin: 0,
+                      "& .MuiSlider-thumb": {
+                        height: 20,
+                        width: 20,
+                        backgroundColor: "#747bff",
+                        "&:hover": {
+                          backgroundColor: "#535bf2",
+                        },
+                        "&.Mui-focusVisible": {
+                          boxShadow:
+                            "0px 0px 0px 8px rgba(116, 123, 255, 0.16)",
+                        },
+                        "&.Mui-active": {
+                          boxShadow:
+                            "0px 0px 0px 14px rgba(116, 123, 255, 0.16)",
+                        },
+                      },
+                      "& .MuiSlider-rail": {
+                        height: 8,
+                        borderRadius: 4,
+                        backgroundColor: "#747bff",
+                      },
+                      "& .MuiSlider-track": {
+                        height: 8,
+                        borderRadius: 4,
+                        backgroundColor: "#747bff",
+                      },
```

Comment:
it is really better to configure things like this in custom theme for avoid code duplication in future (move as much as possible styling to theme, not only for slider), as alternative you could create <MySlider/>, <MyContainedButton/> etc...